scripts/ica_redeem.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
don't think we need to do this in this PR, but we should probably incorporate this into the bats test file

Reply:
good call, added test-74 - working on these now. going to push them to a separate PR